api/android/api/src/main/jni/Android.mk

```diff
@@ -43,6 +41,8 @@ ifeq ($(NNSTREAMER_API_OPTION),single)
 NNS_API_FLAGS += -DNNS_SINGLE_ONLY=1
 endif
 
+include $(NNSTREAMER_ROOT)/jni/nnstreamer.mk
```

Comment:
plz move line 37 under this line (NNSTREAMER_VERSION is defined in nnstreamer.mk)
NNS_API_FLAGS += -DVERSION=\"$(NNSTREAMER_VERSION)\"
NNSTREAMER_VERSION

Reply:
Applied. Thanks!